games/Powermonger/inst/ReadMe

```diff
@@ -47,7 +47,12 @@
 	or	aminet:dev/misc/whdload#?
 	and	aminet:game/patch/
 
-
+	version 1.4 (10.02.2025) done by Paraj:
+	- Use fast RAM to speed up rendering (configurable, defaults to
+	  on if 030+). If the game is now too fast, "CUSTOM2" can be used
+	  to limit the update rate, e.g. CUSTOM2=4 for 12.5Hz if PAL.
+	- Added trainer options to start with all levels and win on retire.
+	- Copy protection dialog is automatically dismissed.
```

Comment:
can you please add:
more icons added
various docs/maps added
new install script

Reply:
Will do